dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -405,13 +413,37 @@ object LagomPlugin extends AutoPlugin {
   }
 
   private def runAllMicroservicesTask: Initialize[Task[Unit]] = Def.taskDyn {
+
     val projects = microservicesProjects.value
-    val filter = ScopeFilter(inProjects(projects: _*))
+    val filter: ScopeFilter = ScopeFilter(inProjects(projects: _*))
+    // check if application.conf is present in service
+    val serviceResourceDirectories = ((unmanagedResourceDirectories in Runtime) ?? Nil).all(filter)
+    //Verifying the presence of the required application.conf file
+    val verifyApplicationConfTask = Def.task {
+
+      val servicesPaths: Seq[File] = serviceResourceDirectories.value.flatten
+
+      val (validProject, notValidProject) = servicesPaths.partition(f => {
+        val r: PathFinder = {
+          f.get * "application.conf"
+        }
+        r.get.nonEmpty
+      })
+      //displays all projects that not contain a application.conf file
+      if (notValidProject.nonEmpty) {
+        notValidProject foreach (f => {
+          val log = state.value.log
+          log.error("File application.conf has been missed in the following service >>> " + f.project)
```

Comment:
Hey Fabrice, no need to apologize, we are all learning! :)

> it's better than what i done. Just one remarks, f is a Seq[File], I think I cannot apply on f the method named project, the method is not recognized. So I propose to display the path where the file is missing.

You could keep track of the project reference associated with the resource directories you are looking at. I think it's nicer to show the project's name, instead of a potentially very long path (which is also noisy, as you, as a user, are likely just interested in knowing what service project failed to start).

> Regarding the point which you speak after from my point of view, the developer must understand what he did. If he don't understand the purpose of the application.conf file, I propose to add a check on the fact that the key :"play.modules.enabled" exists and at least one value has affected to it.
With that, we can display a more complete message which explain the purpose of the file, also we can provide a link to the documentation that talk about this file.

As you suggested, maybe it's best to provide a link to the documentation where it's explained what the application.conf of a service project should contain (the mentioned documentation doesn't exist yet...).

By the way, I'm not sure if checking for the presence of `play.modules.enabled` will work, as that key is actually present in many reference.conf files that get merged together with the main application.conf when the configuration is inspected. There may be ways of looking "just" at the application.conf, but this should be checked.

Regardless, I'm now more of the opinion that a link to the documentation is best. Users need to understand what they are doing. But it would be good to know what happens if a user provides an empty application.conf. Ideally, if an exception is thrown because the service interface isn't bound, I think it could be good to catch the exception, and again point to the user to documentation for what an application.conf should contain.

Though, before you invest more time on this, it would be good to know if @jroper agrees with the approach.